### DISCUSSION TITLE
Enchantment Table seems to not be working

### DISCUSSION THREAD

**Discussion by Dewint** [Geolykt/EnchantmentsPlus#129] [Q&A]
I don't know if this is my problem or if I have some other plugins causing the issue but I have the plugin on Minehut and when ever I try to test the enchants on an enchant table it only displays 4-5 normal commands, all the same, and often one of the enchant boxes are missing

I don't know if there is a fix to this or if it is the plugins issue but please let me know

**Comment by Geolykt** [comment]
Enchantments are only complementary to the vanilla enchantments. That is the enchantment table will not proactively show the enchantments before enchanting - as of now it is impossible to do otherwise.

If you are not getting enchantments at all, create an issue, but it is likely that you are stuck with an unfortunate random number generator seed leading you to believe that enchantments are not possible to obtain.

The fact that one of the enchantment boxes are missing is a bit strange, could you screenshot it so I can take a look at what might be happening?
